src/main/scala/config/MetricsSettings.scala

```diff
@@ -22,7 +22,8 @@ object MetricsSettings {
       CloudWatch(
         config.getBoolean("metrics.cloudwatch.enabled"),
         config.getString("metrics.cloudwatch.name"),
-        config.getInt("metrics.cloudwatch.interval_minutes")
+        config.getInt("metrics.cloudwatch.interval_minutes"),
+        config.getString("metrics.cloudwatch.metric_filter")
```

Comment:
I think we can change the case class to be `metricFilter: Regex` and this line can become :
```
config.getString("metrics.cloudwatch.metric_filter").r
```
The r method is provided implicitly by the Regex object so you can do `"xyz".r` instead of `new Regex("xyz")` like you have below - this way we're validating the value when parsing the config instead of when building the filter.